face_recognition/api.py

```diff
@@ -224,3 +245,19 @@ def compare_faces(known_face_encodings, face_encoding_to_check, tolerance=0.6):
     :return: A list of True/False values indicating which known_face_encodings match the face encoding to check
     """
     return list(face_distance(known_face_encodings, face_encoding_to_check) <= tolerance)
+
+
+def compare_faces_cosine(known_face_encodings, face_encoding_to_check, tolerance=0.85):
```

Comment:
The functions `face_cosine_similarity` and `compare_faces_cosine` sound too similar in function when the second function is intended to filter faces that are similar above a specified threshold. Could we consider changing the name of this function to `find_similar_faces_cosine()`?